osmosis/assetlist.json

```diff
@@ -25759,6 +25759,61 @@
         }
       ]
     },
+    {
```

Comment:
Please only add assets to the very bottom on the existing assets array

Reply:
Nvm, In this case I see how it makes sense to put the TIA straight from Celestia above the TIA the passes through Osmosis